### ISSUE TITLE
Getting stuck while trying to train en_PP-OCRv3_rec.yml

### ISSUE THREAD

**Issue by rioarifando** [PaddlePaddle/PaddleOCR#10250]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- System Environment：Windows
- Related components：Training
- Command Code：python tools/train.py -c configs/rec/PP-OCRv3/en_PP-OCRv3_rec.yml
- Complete Error Message：

I did training using ICDAR2015 data.
so I made a few changes to the en_PP-OCRv3_rec.yml file

from this:
```
Train:
  dataset:
    data_dir: ./train_data/
    label_file_list:
    - ./train_data/train_list.txt

Eval:
  dataset:
    data_dir: ./train_data
    label_file_list:
    - ./train_data/val_list.txt
```
    
 to this:
```
Train:
 dataset:
    data_dir: ./train_data/ic15_data/
    ext_op_transform_idx: 1
    label_file_list: ./train_data/ic15_data/rec_gt_train.txt

Eval:
  dataset:
    data_dir: ./train_data/ic15_data
    label_file_list: ./train_data/ic15_data/rec_gt_test.txt
```
    
I'm getting stuck while trying to train en_PP-OCRv3_rec.yml
program stops during training. this is the last display and nothing else appears after this.
**"[2023/06/27 15:34:25] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations"**

NOTE    
when I use file rec_icdar15_train.yml runs smoothly. however when using en_PP-OCRv3_rec.yml this error occurs.    
    
**Here is the full off error view**
```
[2023/06/27 15:34:25] ppocr INFO: Architecture : 
[2023/06/27 15:34:25] ppocr INFO:     Backbone : 
[2023/06/27 15:34:25] ppocr INFO:         last_conv_stride : [1, 2]
[2023/06/27 15:34:25] ppocr INFO:         last_pool_type : avg
[2023/06/27 15:34:25] ppocr INFO:         name : MobileNetV1Enhance
[2023/06/27 15:34:25] ppocr INFO:         scale : 0.5
[2023/06/27 15:34:25] ppocr INFO:     Head : 
[2023/06/27 15:34:25] ppocr INFO:         head_list : 
[2023/06/27 15:34:25] ppocr INFO:             CTCHead : 
[2023/06/27 15:34:25] ppocr INFO:                 Head : 
[2023/06/27 15:34:25] ppocr INFO:                     fc_decay : 1e-05
[2023/06/27 15:34:25] ppocr INFO:                 Neck : 
[2023/06/27 15:34:25] ppocr INFO:                     depth : 2
[2023/06/27 15:34:25] ppocr INFO:                     dims : 64
[2023/06/27 15:34:25] ppocr INFO:                     hidden_dims : 120
[2023/06/27 15:34:25] ppocr INFO:                     name : svtr
[2023/06/27 15:34:25] ppocr INFO:                     use_guide : True
[2023/06/27 15:34:25] ppocr INFO:             SARHead :
[2023/06/27 15:34:25] ppocr INFO:                 enc_dim : 512
[2023/06/27 15:34:25] ppocr INFO:                 max_text_length : 25
[2023/06/27 15:34:25] ppocr INFO:         name : MultiHead
[2023/06/27 15:34:25] ppocr INFO:     Transform : None
[2023/06/27 15:34:25] ppocr INFO:     algorithm : SVTR
[2023/06/27 15:34:25] ppocr INFO:     model_type : rec
[2023/06/27 15:34:25] ppocr INFO: Eval :
[2023/06/27 15:34:25] ppocr INFO:     dataset :
[2023/06/27 15:34:25] ppocr INFO:         data_dir : ./train_data/ic15_data
[2023/06/27 15:34:25] ppocr INFO:         label_file_list : ['./train_data/ic15_data/rec_gt_test.txt']
[2023/06/27 15:34:25] ppocr INFO:         name : SimpleDataSet
[2023/06/27 15:34:25] ppocr INFO:         transforms :
[2023/06/27 15:34:25] ppocr INFO:             DecodeImage :
[2023/06/27 15:34:25] ppocr INFO:                 channel_first : False
[2023/06/27 15:34:25] ppocr INFO:                 img_mode : BGR
[2023/06/27 15:34:25] ppocr INFO:             MultiLabelEncode : None
[2023/06/27 15:34:25] ppocr INFO:             RecResizeImg :
[2023/06/27 15:34:25] ppocr INFO:                 image_shape : [3, 48, 320]
[2023/06/27 15:34:25] ppocr INFO:             KeepKeys :
[2023/06/27 15:34:25] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2023/06/27 15:34:25] ppocr INFO:     loader :
[2023/06/27 15:34:25] ppocr INFO:         batch_size_per_card : 128
[2023/06/27 15:34:25] ppocr INFO:         drop_last : False
[2023/06/27 15:34:25] ppocr INFO:         num_workers : 4
[2023/06/27 15:34:25] ppocr INFO:         shuffle : False
[2023/06/27 15:34:25] ppocr INFO: Global :
[2023/06/27 15:34:25] ppocr INFO:     cal_metric_during_train : True
[2023/06/27 15:34:25] ppocr INFO:     character_dict_path : ppocr/utils/en_dict.txt
[2023/06/27 15:34:25] ppocr INFO:     checkpoints : None
[2023/06/27 15:34:25] ppocr INFO:     debug : False
[2023/06/27 15:34:25] ppocr INFO:     distributed : False
[2023/06/27 15:34:25] ppocr INFO:     epoch_num : 500
[2023/06/27 15:34:25] ppocr INFO:     eval_batch_step : [0, 2000]
[2023/06/27 15:34:25] ppocr INFO:     infer_img : doc/imgs_words/ch/word_1.jpg
[2023/06/27 15:34:25] ppocr INFO:     infer_mode : False
[2023/06/27 15:34:25] ppocr INFO:     log_smooth_window : 20
[2023/06/27 15:34:25] ppocr INFO:     max_text_length : 25
[2023/06/27 15:34:25] ppocr INFO:     pretrained_model : None
[2023/06/27 15:34:25] ppocr INFO:     print_batch_step : 10
[2023/06/27 15:34:25] ppocr INFO:     save_epoch_step : 3
[2023/06/27 15:34:25] ppocr INFO:     save_inference_dir : None
[2023/06/27 15:34:25] ppocr INFO:     save_model_dir : ./output/v3_en_mobile
[2023/06/27 15:34:25] ppocr INFO:     save_res_path : ./output/rec/predicts_ppocrv3_en.txt
[2023/06/27 15:34:25] ppocr INFO:     use_gpu : True
[2023/06/27 15:34:25] ppocr INFO:     use_space_char : True
[2023/06/27 15:34:25] ppocr INFO:     use_visualdl : False
[2023/06/27 15:34:25] ppocr INFO: Loss :
[2023/06/27 15:34:25] ppocr INFO:     loss_config_list :
[2023/06/27 15:34:25] ppocr INFO:         CTCLoss : None
[2023/06/27 15:34:25] ppocr INFO:         SARLoss : None
[2023/06/27 15:34:25] ppocr INFO:     name : MultiLoss
[2023/06/27 15:34:25] ppocr INFO: Metric :
[2023/06/27 15:34:25] ppocr INFO:     ignore_space : False
[2023/06/27 15:34:25] ppocr INFO:     main_indicator : acc
[2023/06/27 15:34:25] ppocr INFO:     name : RecMetric
[2023/06/27 15:34:25] ppocr INFO: Optimizer :
[2023/06/27 15:34:25] ppocr INFO:     beta1 : 0.9
[2023/06/27 15:34:25] ppocr INFO:     beta2 : 0.999
[2023/06/27 15:34:25] ppocr INFO:     lr :
[2023/06/27 15:34:25] ppocr INFO:         learning_rate : 0.001
[2023/06/27 15:34:25] ppocr INFO:         name : Cosine
[2023/06/27 15:34:25] ppocr INFO:         warmup_epoch : 5
[2023/06/27 15:34:25] ppocr INFO:     name : Adam
[2023/06/27 15:34:25] ppocr INFO:     regularizer :
[2023/06/27 15:34:25] ppocr INFO:         factor : 3e-05
[2023/06/27 15:34:25] ppocr INFO:         name : L2
[2023/06/27 15:34:25] ppocr INFO: PostProcess :
[2023/06/27 15:34:25] ppocr INFO:     name : CTCLabelDecode
[2023/06/27 15:34:25] ppocr INFO: Train :
[2023/06/27 15:34:25] ppocr INFO:     dataset :
[2023/06/27 15:34:25] ppocr INFO:         data_dir : ./train_data/ic15_data/
[2023/06/27 15:34:25] ppocr INFO:         ext_op_transform_idx : 1
[2023/06/27 15:34:25] ppocr INFO:         label_file_list : ['./train_data/ic15_data/rec_gt_train.txt']
[2023/06/27 15:34:25] ppocr INFO:         name : SimpleDataSet
[2023/06/27 15:34:25] ppocr INFO:         transforms :
[2023/06/27 15:34:25] ppocr INFO:             DecodeImage :
[2023/06/27 15:34:25] ppocr INFO:                 channel_first : False
[2023/06/27 15:34:25] ppocr INFO:                 img_mode : BGR
[2023/06/27 15:34:25] ppocr INFO:             RecConAug :
[2023/06/27 15:34:25] ppocr INFO:                 ext_data_num : 2
[2023/06/27 15:34:25] ppocr INFO:                 image_shape : [48, 320, 3]
[2023/06/27 15:34:25] ppocr INFO:                 max_text_length : 25
[2023/06/27 15:34:25] ppocr INFO:                 prob : 0.5
[2023/06/27 15:34:25] ppocr INFO:             RecAug : None
[2023/06/27 15:34:25] ppocr INFO:             MultiLabelEncode : None
[2023/06/27 15:34:25] ppocr INFO:             RecResizeImg :
[2023/06/27 15:34:25] ppocr INFO:                 image_shape : [3, 48, 320]
[2023/06/27 15:34:25] ppocr INFO:             KeepKeys :
[2023/06/27 15:34:25] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2023/06/27 15:34:25] ppocr INFO:     loader :
[2023/06/27 15:34:25] ppocr INFO:         batch_size_per_card : 128
[2023/06/27 15:34:25] ppocr INFO:         drop_last : True
[2023/06/27 15:34:25] ppocr INFO:         num_workers : 4
[2023/06/27 15:34:25] ppocr INFO:         shuffle : True
[2023/06/27 15:34:25] ppocr INFO: profiler_options : None
[2023/06/27 15:34:25] ppocr INFO: train with paddle 2.4.2 and device Place(gpu:0)
[2023/06/27 15:34:25] ppocr INFO: Initialize indexs of datasets:['./train_data/ic15_data/rec_gt_train.txt']
[2023/06/27 15:34:25] ppocr INFO: Initialize indexs of datasets:['./train_data/ic15_data/rec_gt_test.txt']
W0627 15:34:25.060976 21124 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 11.7, Runtime API Version: 11.7
W0627 15:34:25.063975 21124 gpu_resources.cc:91] device: 0, cuDNN Version: 8.9.
[2023/06/27 15:34:25] ppocr INFO: train dataloader has 34 iters
[2023/06/27 15:34:25] ppocr INFO: valid dataloader has 17 iters
[2023/06/27 15:34:25] ppocr INFO: train from scratch
[2023/06/27 15:34:25] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations
(.venv) C:\code\project2_paddle\PaddleOCR>
```

Got stopped in this process. no error. no information

**Comment by livingbody** [comment]
what's about you batch size ?
check your gpu env

**Comment by sumitkutty** [comment]
Facing the same solution. Did you find out why this was happening? @rioarifando

**Comment by UserWangZz** [comment]
This issue has not been updated for a long time. This issue is temporarily closed and can be reopened if necessary.

**Comment by hypercell-ahmed-mehanna** [comment]
Dear @UserWangZz,

I'm facing the same issue, I tried to train mode using GPU and CPU with different batch size and iterations, but still it is stuck at the same point.

**Comment by UserWangZz** [comment]
> Dear @UserWangZz,
> 
> I'm facing the same issue, I tried to train mode using GPU and CPU with different batch size and iterations, but still it is stuck at the same point.

how about the version of paddlepaddle and paddleocr?

**Comment by saurabhmali1** [comment]
Is it resolved? I am facing same issues with det model

**Comment by suyeon1113** [comment]
ch_PP-OCRv3_rec_distillation.yml 에서도 같은 에러를 겪고 있습니다

**Comment by Unlicensed-driver-ljx** [comment]
各位大哥解决了吗

**Comment by Unlicensed-driver-ljx** [comment]
放一个解决办法出来把

**Comment by yesungKo** [comment]
Is it resolved?

**Comment by rioarifando** [comment]
change the batch size it will solve the problem

**Comment by phongbmatsjsc** [comment]
same problem. changed the batch_size as well.
